evalme/tests/test_text.py

```diff
@@ -0,0 +1,54 @@
+import pytest
```

Comment:
Do we cover cases where one xpath node is splitted onto 2 blocks? like `block1 <b><i> block2 </i></b>` in the initial issue

Reply:
We don't cover these cases as evalution method covers only text. 
Initial fix of the issue ignores xpath tree as we don't have information about it.

Reply:
sure, but at least we can nail down it into the text field just to be sure we won't miss anything later, when coming up with a tree-aware solution